docs/src/logical_replication.md

```diff
@@ -0,0 +1,444 @@
+# Logical Replication
+
+PostgreSQL extends its replication capabilities beyond physical replication,
+which operates at the level of exact block addresses and byte-by-byte copying,
+by offering [logical replication](https://www.postgresql.org/docs/current/logical-replication.html).
+Logical replication replicates data objects and their changes based on a
+defined replication identity, typically the primary key.
+
+Logical replication uses a publish-and-subscribe model, where subscribers
+connect to publications on a publisher node. Subscribers pull data changes from
+these publications and can re-publish them, enabling cascading replication and
+complex topologies.
+
+This flexible model is particularly useful for:
+
+- Online data migrations
+- Live PostgreSQL version upgrades
+- Data distribution across systems
+- Real-time analytics
+- Integration with external applications
+
+!!! Info
+    For more details, examples, and limitations, please refer to the
+    [official PostgreSQL documentation on Logical Replication](https://www.postgresql.org/docs/current/logical-replication.html).
+
+**CloudNativePG** enhances this capability by providing declarative support for
+key PostgreSQL logical replication objects:
+
+- **Publications** via the `Publication` resource
+- **Subscriptions** via the `Subscription` resource
+
+## Publications
+
+In PostgreSQL's publish-and-subscribe replication model, a
+[**publication**](https://www.postgresql.org/docs/current/logical-replication-publication.html)
+is the source of data changes. It acts as a logical container for the change
+sets (also known as *replication sets*) generated from one or more tables within
+a database. Publications can be defined on any PostgreSQL 10+ instance acting
+as the *publisher*, including instances managed by popular DBaaS solutions in the
+public cloud. Each publication is tied to a single database and provides
+fine-grained control over which tables and changes are replicated.
+
+For publishers outside Kubernetes, you can [create publications using SQL](https://www.postgresql.org/docs/current/sql-createpublication.html)
+or leverage the [`cnpg publication create` plugin command](kubectl-plugin.md#logical-replication-publications).
+
```

Comment:
I'm not sure I'm getting this. In this paragraph we're on the topic of publishers outside Kubernetes. So does it make sense to offer the possibility of using the kubectl plugin command `publication create`? Is that better in a separate paragraph?

Reply:
`publication create` works on source databases outside Kubernetes as long as the destination Cluster has a usable definition for the source in the external clusters.